Dockerfile

```diff
@@ -37,21 +17,32 @@ WORKDIR /code
 
 # Install Python dependencies
 RUN pip install poetry
-COPY pyproject.toml poetry.lock /code/
+COPY pyproject.toml poetry.lock README.md /code/
```

Comment:
We shouldn't copy in README

Reply:
The README.md is required by pyproject.toml line 6. We'd need to remove that line too if we don't want the README in the image.

Reply:
Just do 'touch README.md' instead then I think. There's a flag to override this but I don't recall.